search/2016-09-01/swagger/searchindex.json

```diff
@@ -0,0 +1,289 @@
+        },
+        "highlight": {
+          "type": "string",
+          "description": "The comma-separated list of field names to use for hit highlights. Only searchable fields can be used for hit highlighting."
+        },
+        "highlightPostTag": {
+          "type": "string",
+          "description": "A string tag that is appended to hit highlights. Must be set with HighlightPreTag. Default is &lt;/em&gt;."
+        },
+        "highlightPreTag": {
+          "type": "string",
+          "description": "A string tag that is prepended to hit highlights. Must be set with HighlightPostTag. Default is &lt;em&gt;."
+        },
+        "minimumCoverage": {
+          "type": "number",
+          "format": "double",
+          "description": "A number between 0 and 100 indicating the percentage of the index that must be covered by a search query in order for the query to be reported as a success. This parameter can be useful for ensuring search availability even for services with only one replica. The default is 100."
+        },
+        "orderby": {
+          "x-ms-client-name": "OrderBy",
+          "type": "string",
+          "description": "The comma-separated list of OData $orderby expressions by which to sort the results. Each expression can be either a field name or a call to the geo.distance() function. Each expression can be followed by asc to indicate ascending, and desc to indicate descending. The default is ascending order. Ties will be broken by the match scores of documents. If no OrderBy is specified, the default sort order is descending by document match score. There can be at most 32 Orderby clauses."
+        },
+        "queryType": {
+          "$ref": "#/definitions/QueryType",
+          "description": "Gets or sets a value that specifies the syntax of the search query. The default is 'simple'. Use 'full' if your query uses the Lucene query syntax."
+        },
+        "scoringParameters": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of parameter values to be used in scoring functions (for example, referencePointParameter) using the format name:value. For example, if the scoring profile defines a function with a parameter called 'mylocation' the parameter string would be \"mylocation:-122.2,44.8\"(without the quotes)."
+        },
+        "scoringProfile": {
+          "type": "string",
+          "description": "The name of a scoring profile to evaluate match scores for matching documents in order to sort the results."
+        },
+        "search": {
+          "externalDocs": {
+            "url": "https://docs.microsoft.com/rest/api/searchservice/Simple-query-syntax-in-Azure-Search"
+          },
+          "type": "string",
+          "description": "A full-text search query expression; Use null or \"*\" to match all documents."
+        },
+        "searchFields": {
```

Comment:
@dsgouda Hindsight is 20/20. This API has been in production for several years; We are not going to change it now.